### DISCUSSION TITLE
Copilot for Business: As an Administrator I would like to analyze usage

### DISCUSSION THREAD

**Discussion by fstienhans-lg** [community/community#56411] [Copilot Conversations] [answered]
1. Who is using it and who is not (shelf-ware question)
2. How much is it being used.
3. How many proposals by Copilot are being used
4. Average Number of Generated Bytes per Used Proposal
5. List of Used Proposals and their developer inputs

I have looked for this information. As I do not find it, I am requesting this feature.

**Comment by Akash1134** [comment]
Hi, 

Thanks for your feedback @FrankStienhans3 

Currently, extracting such analytics from GitHub Copilot is not possible. This could be a capability our team would love to consider for a later release.

I have added the relavent label to the discussion for visibility amongst other feature requests. 🙂 

Thanks!

**Reply by askpatrickw** [reply, marked as answer]
this is a dupe of https://github.com/orgs/community/discussions/54691 which has more votes
